codeforamerica/vita-min
# Add mTLS certs for AZ, MD, NC, NJ, ID and Demo site

## Link to pivotal/JIRA issue
- e.g. [https://codeforamerica.atlassian.net/browse/FYST-496](https://codeforamerica.atlassian.net/browse/FYST-496)
## Is PM acceptance required? (delete one)
- No - merge after code review approval
## What was done?
- Added mTLS certs in credentials file for NC, NJ, ID and demo
## How to test?
- Go into the credentials file for production and demo and make sure its formatted correctly
- can use this command to see the difference between main and this branch's credentials files `bin/gyr credentials_diff --environment production --base main`
